packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -2112,20 +2112,13 @@ describe('RichTextEditor', () => {
             ).toEqual(['1']);
         });
 
-        // TODO: Once the rich text validator added for duplicate configuration elements, below test case should be updated
-        it('adding two mention configuration elements in the same editor should render as mention node', async () => {
+        it('adding two mention configuration elements in the same editor should render as anchor element', async () => {
             element.setMarkdown('<user:1>');
             await appendUserMentionConfiguration(element);
             await appendUserMentionConfiguration(element);
 
-            expect(pageObject.getMarkdownRenderedTagNames()).toEqual([
-                'P',
-                `${richTextMentionUsersViewTag}`.toUpperCase(),
-                'BR'
-            ]);
-            expect(
-                pageObject.getEditorMentionViewAttributeValues('mention-label')
-            ).toEqual(['1']);
+            expect(pageObject.getEditorTagNames()).toEqual(['P', 'A']);
```

Comment:
Why is this test not using the `anchorTag` approach as done in the rich-text-editor tests? Are we using different elements between the two?

Reply:
Yes, we are using `anchorTag` in the viewer but not in the editor due to the issues listed in https://github.com/ni/nimble/issues/1516. These issues are majorly related to rendering `nimble-anchor` in `contenteditable` div like the rich text editor. Whereas in the viewer, it does not have any issues in rendering so used the same. The goal is to use `nimble-anchor` in both components and in the editor, we will replace `<a>` with `nimble-anchor` once the linked issue is resolved.